.github/workflows/actions.yml

```diff
@@ -55,3 +58,9 @@ jobs:
           coveralls --service=github
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Test benchadapt
+        run: |
+          pytest -vvvv benchadapt/python/tests
+      - name: Test benchrun
+        run: |
+          pytest -vvvv benchrun/python/tests
```

Comment:
I lol every time. Technically,
<img width="507" alt="image" src="https://user-images.githubusercontent.com/16600275/196737028-93ac192e-849d-4b3d-b14b-4bb50d455cde.png">
but you're free to do whatever you want!

Reply:
yeah we can probably take this back to `-vv`; I think I got this in my brain when I was looking for the debug flag